tools/test/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+load("@bazel_skylib//rules:diff_test.bzl", "diff_test")
+load("@aspect_bazel_lib//lib:jq.bzl", "jq")
+
+jq(
+    name = "final_package_json",
+    # buildifier: do not sort
+    srcs = [
+        "root_package.json",
+        "project_package.json",
+    ],
+    args = [
+        "--slurp",
+    ],
+    filter_file = "//tools:package_json_filter.txt",
```

Comment:
nit: a more verbose name on this file would be good. something like `package_json_release_filter.jq`? I made up the `.jq` extension but it seems reasonable